app/assets/javascripts/barcode_items.js.erb

```diff
@@ -45,9 +45,28 @@ $(document).ready(function() {
   */
   function fill_fields_with_barcode_results(data) {
     line_item = $(data['src']).closest('.nested-fields');
-    $(line_item).find('input[type=number]').val(data['quantity']);
+      line_item_quantity = data['quantity'];
+      $('input[type=text]').each(function () {
+          if (data['src'] != this && data['value'] == this.value) {
+              line_item = this.closest('.nested-fields');
+              if ($(line_item).attr('scanned_more_than_two_times') != undefined) {
+                  prompt_value = prompt('Enter quantity', line_item_quantity);
```

Comment:
I think I'd chance the language of this from `Enter quantity` to `Enter quantity of additional diapers`